src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
+import GroupsController from '@controllers/groupe.controller';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { Routes } from '@interfaces/routes.interface';
+import validationMiddleware from '@middlewares/validation.middleware';
+
+class GroupsRoute implements Routes {
+  public path = '/groups';
+  public router = Router();
+  public groupsController = new GroupsController();
+
+  constructor() {
+    this.initializeRoutes();
+  }
+
+  private initializeRoutes() {
+    this.router.get(`${this.path}`, this.groupsController.getGroups);
```

Comment:
```suggestion
    this.router.get(this.path, this.groupsController.getGroups);
```

No need to create a new string